pkg/security/security.go

```diff
@@ -402,6 +402,16 @@ func CurrentNamespace() string {
 	return namespace
 }
 
+// CurrentNamespaceOrError returns the namespace of this workload. If current
+// Namespace is not found, error.
+func CurrentNamespaceOrError() (string, error) {
+	namespace, ok := os.LookupEnv("NAMESPACE")
+	if !ok {
```

Comment:
What if NAMESPACE is set but empty?

Reply:
In this case, we would end up with an error downstream. As you point out, it is probably better to error in this case. I think a separate error message would be more proper.